django_freeradius/base/models.py

```diff
@@ -819,3 +820,36 @@ def _remove_files(self):
         path = getattr(self, strategy_filemap.get(self.strategy)).path
         if os.path.isfile(path):
             os.remove(path)
+
+
+class AbstractRadiusToken(AutoUsernameMixin, models.Model):
+    key = models.CharField(_("Key"), max_length=40, primary_key=True)
+
+    username = models.CharField(verbose_name=_('username'),
+                                max_length=64,
+                                db_index=True,
+                                blank=True)
+
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
```

Comment:
I think we should opt for a `OneToOneField`, so we make sure that each user can have max 1 token.
Instead of creating a new one, we'll have to update the existing one.
Otherwise the DB will get full of old tokens and we should implement a clean up operation, but we can avoid that by making sure we will always have max one token per user.

Reply:
Yes, I just realized it should be one to one field. I'll fix it.